osc/iterm2.go

```diff
@@ -5,11 +5,14 @@ import (
 	"encoding/base64"
 	"errors"
 	"fmt"
+	"image/png"
 	"io"
 	"os"
 	"sync"
 	"time"
 
+	"github.com/mattn/go-sixel"
```

Comment:
Can we isolate sixel support in a `sixel.go` file and have public methods that are used for both terms in a `common.go` or something?